com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/HeliumUnityAdWrapper.kt

```diff
@@ -0,0 +1,204 @@
+package com.chartboost.heliumsdk.unity
+
+import android.app.Activity
+import android.graphics.Color
+import android.util.DisplayMetrics
+import android.util.Log
+import android.view.Gravity
+import android.view.View
+import android.view.ViewGroup
+import android.widget.RelativeLayout
+import com.chartboost.heliumsdk.ad.HeliumAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd
+import com.chartboost.heliumsdk.ad.HeliumBannerAd.HeliumBannerSize
+import com.chartboost.heliumsdk.ad.HeliumFullscreenAd
+import com.chartboost.heliumsdk.ad.HeliumRewardedAd
+import com.unity3d.player.UnityPlayer
+
+class HeliumUnityAdWrapper(private val ad: HeliumAd) {
+    //
+    private val activity: Activity? = UnityPlayer.currentActivity
+
+    /**
+     * This is the container for the banner.
+     * We need a relative layout to position the banner in one of the 7 possible positions.
+     * HeliumBannerAd is just a ViewGroup.
+     * */
+    private var bannerLayout: RelativeLayout? = null
+    private var startedLoad = false
+
+    fun load() {
+        HeliumUnityBridge.runTaskOnUiThread {
+            ad.load()
+            startedLoad = true
```

Comment:
so this isn't part of the direct conversion, but startedLoad never gets flipped back to false.
If you load then clearLoaded(), we still pretend a load has started. I guess this is mostly to check that we don't actually check the underlying helium ad if we never loaded? Not sure... but just wondering.

Reply:
Yes, this is legacy code and should be removed without issues.